kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/checks/ConstructorArgCountCheck.kt

```diff
@@ -4,14 +4,17 @@ import godot.entrygenerator.model.SourceFile
 import godot.entrygenerator.utils.Logger
 
 class ConstructorArgCountCheck(logger: Logger, sourceFiles: List<SourceFile>): BaseCheck(logger, sourceFiles) {
-    override fun execute() {
+    override fun execute(): Boolean {
+        var hasIssue = false
         sourceFiles
             .flatMap { it.registeredClasses }
             .flatMap { it.constructors }
             .forEach { registeredConstructor ->
                 if (registeredConstructor.parameters.size > 5) {
```

Comment:
I would add a comment here for `5`, to say it is linked to `transfer_context.cpp` where we have:

```cpp
const int VARIANT_ARG_MAX {5};
```